build.assets/Makefile

```diff
@@ -0,0 +1,46 @@
+# This Makefile is used by CI/CD builds.
+#
+# Prerequisites:
+#     - Docker 1.9.1 or newer
+#     - You must be a part of 'docker' group to use Docker without sudo
+#
+
+.DEFAULT_GOAL := ci
+
+OPS_URL :=
+OPS_KEY :=
+TAG :=
+TELE_FLAGS :=
+CWD := $(shell dirname $(realpath $(lastword $(MAKEFILE_LIST))))
+
+
+.PHONY: ci
+ci: tele-login
+	$(eval TMP := $(shell mktemp -d))
+	mkdir -p $(TMP)/resources/
+	cp $(CWD)/resources/* $(TMP)/resources/
+	find $(TMP)/resources/ -type f -name '*.yaml' -exec sed -i 's|XXX_PROVISIONER_VERSION_XXX|$(TAG)|' {} \;
```

Comment:
for this use case let's use tele's `--set-image` flag instead of sed"

https://github.com/gravitational/gravity/blob/master/tool/tele/main.go#L70

Reply:
```
IMPORT_IMAGE_FLAGS := --set-image=imagename-1:$(VERSION) \
	--set-image=imagename-2:$(VERSION)

TELE_BUILD_OPTIONS := --insecure \
                --repository=$(OPS_URL) \
                --name=$(NAME) \
                --version=$(VERSION) \
                --glob=**/*.yaml \
                --ignore=".git" \
                --ignore="images" \
                --output="$(TMP)/installer.tar"
                $(IMPORT_IMAGE_FLAGS)
tele build $(TELE_BUILD_OPTIONS) resources/app.yaml
```